packages/components/src/components/modal/modal.css

```diff
@@ -30,29 +30,30 @@
     -spacing-56 !== --telekom-spacing-unrelated-sm
   */
   --size-window-small: calc(
-    (6 * var(--telekom-spacing-unrelated-sm, 3.5rem)) +
-      (5 * var(--telekom-spacing-related-sm))
+    (6 * var(---telekom-spacing-unit-x14, 3.5rem)) +
+      (5 * var(--telekom-spacing-unit-x8))
   );
   --size-window-default: calc(
-    (8 * var(--telekom-spacing-unrelated-sm, 3.5rem)) +
-      (7 * var(--telekom-spacing-related-sm))
+    (8 * var(--telekom-spacing-unit-x14, 3.5rem)) +
+      (7 * var(--telekom-spacing-unit-x8))
   );
   --size-window-large: calc(
-    (12 * var(--telekom-spacing-unrelated-sm, 3.5rem)) +
-      (11 * var(--telekom-spacing-related-sm))
+    (12 * var(--telekom-spacing-unit-x14, 3.5rem)) +
+      (11 * var(--telekom-spacing-unit-x8))
   );
 
-  --spacing-x-header: var(--telekom-spacing-grouped-lg);
-  --spacing-y-header: var(--telekom-spacing-grouped-lg);
+  --spacing-x-header: var(--telekom-spacing-unit-x6);
+  --spacing-y-header: var(--telekom-spacing-unit-x6);
 
+  /* FIXME:TOKEN --telekom-color-ui-border-standard is too dark */
   --border-bottom-header-has-scroll: var(--telekom-line-weight-standard) solid
-    var(--telekom-color-ui-light); /* could also be --telekom-color-ui-subtle ? */
+    var(--telekom-color-ui-general-subtle, var(--telekom-color-ui-subtle));
```

Comment:
`--telekom-color-ui-general-subtle` is this a new token? not seeing it here https://unpkg.com/@telekom/design-tokens@1.0.0-alpha.5/dist/telekom/css/telekom-design-tokens.all.css . Anyway, if it is, maybe we have to update some other components to use this as well?